.zuul.d/jobs.yaml

```diff
@@ -0,0 +1,29 @@
+---
+- job:
+    name: ansible-navigator-tox-py38
+    parent: ansible-buildset-registry-consumer
+    dependencies:
+      - ansible-buildset-registry
+    timeout: 5400
+    vars:
+      container_command: podman
+      tox_extra_args: -vv --skip-missing-interpreters false
```

Comment:
:heart:

Reply:
This will be the default in ~2 weeks anyway but why to wait. I can remove it after i merge it in zuul-jobs.